packages/discord.js/src/client/WebhookClient.js

```diff
@@ -1,119 +0,0 @@
-'use strict';
-
-const { DiscordjsError, ErrorCodes } = require('../errors/index.js');
-const { Webhook } = require('../structures/Webhook.js');
-const { parseWebhookURL } = require('../util/Util.js');
-const { BaseClient } = require('./BaseClient.js');
-
-/**
- * The webhook client.
- *
- * @implements {Webhook}
- * @extends {BaseClient}
- */
-class WebhookClient extends BaseClient {
-  /**
-   * Represents the credentials used for a webhook in the form of its id and token.
-   *
-   * @typedef {Object} WebhookClientDataIdWithToken
-   * @property {Snowflake} id The webhook's id
-   * @property {string} token The webhook's token
-   */
-
-  /**
-   * Represents the credentials used for a webhook in the form of a URL.
-   *
-   * @typedef {Object} WebhookClientDataURL
-   * @property {string} url The full URL for the webhook
-   */
-
-  /**
-   * Represents the credentials used for a webhook.
-   *
-   * @typedef {WebhookClientDataIdWithToken|WebhookClientDataURL} WebhookClientData
-   */
-
-  /**
-   * Options for a webhook client.
-   *
-   * @typedef {Object} WebhookClientOptions
-   * @property {MessageMentionOptions} [allowedMentions] Default value for {@link BaseMessageOptions#allowedMentions}
-   * @property {RESTOptions} [rest] Options for the REST manager
-   */
-
-  /**
-   * @param {WebhookClientData} data The data of the webhook
-   * @param {WebhookClientOptions} [options] Options for the webhook client
-   */
-  constructor(data, options) {
-    super(options);
-    Object.defineProperty(this, 'client', { value: this });
-    let { id, token } = data;
-
-    if ('url' in data) {
-      const parsed = parseWebhookURL(data.url);
-      if (!parsed) {
-        throw new DiscordjsError(ErrorCodes.WebhookURLInvalid);
```

Comment:
This error is used nowhere else, should probably be removed from `/errors/ErrorCodes.js` and `/errors/Messages.js` and typings now.